src/mgmt/cli_mgmt.c

```diff
@@ -280,6 +281,16 @@ struct dc_mgmt_psr {
 	__rc;						\
 })
 
+static void
+put_attach_info(int npsrs, struct dc_mgmt_psr *psrs)
```

Comment:
I assume this name was chosen to be the converse of `get_attach_info()`... While the symmetry is nice, it's a little misleading. As a reader of code, I'd prefer not to have to go look at the implementation of a function to figure out what it really does. I suggest `free_psr_list()` for the "boringly descriptive" angle.

Reply:
This function was pre-existing as-is name and all.  I had to move it up north so that it could be called by my function after it.  Adding a forward declaration would work, but broke the style of the rest of the file so I kept the style and just moved it.